### DISCUSSION TITLE
about the work of the library

### DISCUSSION THREAD

**Discussion by Zont1k** [BundledSticksInkorperated/Discore#100] [Q&A] [answered]
If you don't mind, i'm ready to create a discord server and wiki page for this library.

**Comment by Francessco121** [comment, marked as answer]
Sorry, but I don't think a Discord server would be very helpful for Discore. This isn't a popular library (and that's ok), I'd rather keep discussions and questions here on GitHub.

Also, there's already a wiki: https://github.com/Francessco121/Discore/wiki

**Reply by Zont1k** [reply]
Thanks for answering.
From my own experience, i can say that the discord guild would be very helpful, because if users had any questions about the library, they could immediately answer them. And thus, library can become more popular.

**Reply by Francessco121** [reply]
I'd still rather keep discussions here for a few reasons: easier to moderate (if ever needed) and questions/answers are google-able.

Regarding popularity, there really isn't a reason for Discore to be popular at the moment. Compared to the other .NET Discord libraries, Discore is very behind in features. Unfortunately, I don't have the time lately to invest in keeping up the with latest API additions aside from maintenance. And because of that, I certainly don't want a whole Discord server to also worry about. I appreciate that you sound like you're interested in maintaining the server but ultimately I'm gonna end up needing to answer questions anyway, so again it's just easier for me if that stays on GitHub.

I hope you can understand.

**Reply by Zont1k** [reply]
No problem. My job is to offer. Have a nice day
